internal/registry/operator_test.go

```diff
@@ -24,8 +24,8 @@ import (
 )
 
 func TestIsOperatorCertified(t *testing.T) {
-	name := "zoperator.v0.3.6"
```

Comment:
zoperator can be still used, as it will appear in the catalog as soon the api is fixed, so this change is not needed.